File: Kattis/C++/1_Easy/kikiboba.cpp
#include <iostream>
#include <string>
#include <algorithm>
using namespace std;

int main() {
    string word;
    cin >> word;

    int countB = count(word.begin(), word.end(), 'b');
    int countK = count(word.begin(), word.end(), 'k');

    if (countB > countK) {
        cout << "boba" << endl;
    } 
    else if (countK > countB) {
        cout << "kiki" << endl;
    } 
    else if (countB == countK && countB != 0) {
        cout << "boki" << endl;
    } 
    else {
        cout << "none" << endl;
    }

    return 0;
}

File: kattis/python/1_Easy/kikiboba.py
message = input()

count_b = message.count('b')
count_k = message.count('k')

if count_b > count_k:
    print("boba")
elif count_k > count_b:
    print("kiki")
elif count_b == count_k and count_b > 0:  
    print("boki")
else:
    print("none")
